os/lib/memb.c

```diff
@@ -62,10 +62,10 @@ memb_alloc(struct memb *m)
 
   for(i = 0; i < m->num; ++i) {
     if(m->count[i] == 0) {
-      /* If this block was unused, we increase the reference count to
+      /* If this block was unused, we set the count flag to
 	 indicate that it now is used and return a pointer to the
 	 memory block. */
-      ++(m->count[i]);
+      m->count[i] = 1;
```

Comment:
Then, I think it'd be better to rename the member, `count` to `used`.

Now, I'm not sure why the type of the member is `char`... This one may be changed to `bool`? With these changes, this line would be:
```
m->used[i] = true;
```

Reply:
Done. Thanks for the review